app/src/org/commcare/views/QuestionsView.java

```diff
@@ -347,7 +347,15 @@ public void setFocus(Context context, int indexOfLastChangedWidget) {
     }
 
     private void scrollToWidget(final QuestionWidget widget) {
-        new Handler().post(() -> QuestionsView.this.scrollTo(0, widget.getTop()));
+        new Handler().post(() -> {
+            String widgetTextId = "None";
+            try {
+                widgetTextId = widget.getPrompt().getQuestion().getTextID();
+            } catch (Exception ignore) {}
+            Logger.log(LogTypes.SOFT_ASSERT, "Attempting to scroll to widget: " + widgetTextId);
+
+            QuestionsView.this.scrollTo(0, widget.getTop());
```

Comment:
@avazirna you can instead catch the exception in this call and rethrow with the additional details, that way we will only log this in case of errrors, otherwise I am afraid this is a rather high frequency log that we will want to avoid logging.

Reply:
@shubham1g5 I haven't been able to reproduce, but one common factor between the forms where this issue is happening is the app callout question, so I'm trying to assess whether it's related. Within QuestionsView there is the `restoreFocusToQuestionThatCalledOut()` which triggers the logic to `scrollToWidget()`, so the previous log was to assess whether this is the cause and what's the widget causing the issue.